SEED-platform/seed
# Fixed `Clear all filters` button

https://www.pivotaltracker.com/n/projects/1488476/stories/126535275

#### What's this PR do?
The grid menu's `Clear all filters` button no longer removes filter conditions when clicked, so custom text and number filters persist.

#### How should this be manually tested?
Use the `DC BlueSky Test` organization.

Filter a column using the SEED filtering syntax (e.g. `!null`).  Click the grid menu on the bluesky page, click `Clear all filters`, and then try using the same filter to verify that it still works.
